.github/workflows/test.yml

```diff
@@ -37,4 +37,45 @@ jobs:
         run: dotnet build --no-restore -c Debug
       - name: Test solution [Debug]
         working-directory: src
-        run: dotnet test --no-restore
+        run: dotnet test --no-restore -p:CollectCoverage=true -p:CoverletOutputFormat=lcov -p:Include=[GraphQL-Parser]* -p:CoverletOutput=../../${{ matrix.os }}.lcov.info
+      - name: Convert coverage report to clover format
+        uses: danielpalme/ReportGenerator-GitHub-Action@4.7.1
+        with:
+          reports: '${{ matrix.os }}.lcov.info'
+          targetdir: '.'
+          reporttypes: 'Clover'
+          tag: '${{ github.run_number }}_${{ github.run_id }}'
+      - name: Monitor coverage
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+        uses: slavcodev/coverage-monitor-action@1.2.0
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          clover_file: "Clover.xml"
+          threshold_alert: 80
+          threshold_warning: 90
```

Comment:
Can you explain what this does?  It would seem that it triggers some type of alert or failure if code coverage drops below a certain percentage?  Isn't that a bit pointless?  I mean, we really want to know the amount of coverage for the code modified in the PR, not as a global percentage.  It's not like people are going to change 10% of the codebase.  Or is that what this does?

Reply:
https://github.com/graphql-dotnet/parser/pull/89#issuecomment-719935986

This is a comment showing the percentage of code coverage by tests.

Reply:
This is nothing more than supplementary information that may be useful for consideration. If the PR is such that the coverage drops significantly, then this is a reason to think.

Reply:
Hmm ok

Reply:
deleted